frame/contracts/src/wasm/runtime.rs

```diff
@@ -335,8 +335,8 @@ impl RuntimeCosts {
 				.hash_blake2_128
 				.saturating_add(s.hash_blake2_128_per_byte.saturating_mul(len.into())),
 			EcdsaRecovery => s.ecdsa_recover,
-			ChainExtension(amount) => amount,
-			CallRuntime(weight) => weight.ref_time(),
+			ChainExtension(amount) => Weight::from_ref_time(amount),
```

Comment:
I think we should rather just change the variant to `ChainExtension(Weight)`.  This would be more consistent with the rest of the code.